tuning/sft_trainer.py

```diff
@@ -250,6 +250,10 @@ def train(
     )
 
     if data_args.chat_template:
+        # TODO: passing "/n" through cli causes parsing issues,
```

Comment:
should this be part of this PR or a separate PR?

Reply:
Its fine to fix it as part of this PR as it touches most of the chat template functionality @ashokponkumar 
Its called out as a temp fix via the `TODO` we can get to a better solution later.

Reply:
sure, lets go ahead then.